src/content/docs/tutorial-reporting-data/capture-the-right-data.mdx

```diff
@@ -0,0 +1,121 @@
+---
+title: "Capture the right data with New Relic" 
+metaDescription: "This guide helps you set up the best telemetry possible to describe the runtime operation of your services."
+redirects:
+  - /docs/apm/agents/manage-apm-agents/agent-data/custom-instrumentation
+  - /docs/new-relic-solutions/observability-maturity/operational-efficiency/sc-implementation-guide
+  - /docs/new-relic-solutions/observability-maturity/operational-efficiency/service-characterization-optimize-telemetry-data-guide/
+---
+
+Modern monitoring systems can give you important insight into the details of a service when they report the right data. Distributed trace, byte code, or script instrumentation allows teams to quickly collect detailed service telemetry. Unfortunately, operations teams usually aren't in the best position to evaluate the quality of the telemetry they get.
+
+Revealing improperly instrumented services to users creates a situation that can put customer satisfaction at risk. This burn-in period is difficult to escape as new features can come from code bases without a strong link between software delivery and observability programs. Service instrumentation planning is the approach used to describe a single service runtime through telemetry, and this guide focuses on the metrics of your application's code as well as external measurements through synthetic testing. 
+
+You're a good candidate for using this guide if any of the following are true:
+* Your development teams are disconnected from production observability design.
+* You have new services/capabilities that run in production and before fully establishing telemetry and alerting.
+* You need to provide additional business context to your instrumentation to improve diagnosis and business KPI measurement.
+* You employ a highly customized or proprietary software framework.
+* Your service is under active development. Legacy services, and services built from commercial-off-the-shelf platforms, tend to be better served with generic instrumentation options.
+
+## Understand the benefits [#understand-benefits]
+
+Involving developers in this process has the following benefits:
```

Comment:
I think I get a bit lost here. The header is about understanding the benefits, but now I'm confused as to what to the benefits of _what_  we're talking about.

This is the first time "involving developers" is mentioned and "this process" is a bit vague. I think this is an easy fix by talking more directly about what the "process" is and why developers are involved earlier in the doc